src/constants.ts

```diff
@@ -43,20 +43,20 @@ export type ENV_FILTER_PROPS =
 
 export const Env = {
     gaga: {
-        Aurora: "Aurora\\.\\[1\\]AuroraSecondary$",
-        Lightning: "[3\\]LightingWithTarget$",
-        solidLaser: "[0\\]FrontLaserL$",
-        directionalLight: "DirectionalLights\\.\\[0\\]DirectionalLightFront$"
+        Aurora: "AuroraSecondary$",
+        Lightning: "1L\\.\\[\\d+\\]\\w+\\.\\[\\d+\\]LightningWithTarget$",
```

Comment:
This one ended up a bit longer than I had hoped. The "LightningWithTarget" can probably be shortened to something like "\w+With\w+t$" or maybe even "\w{19}$" or something like that.